cypress/e2e/patient_spec/encounter-notes.cy.ts

```
@@ -0,0 +1,78 @@
+    cy.wait(1000);
+
+    // second thread
+    cy.contains("button", "New Thread").click();
+    cy.get("input[placeholder='Enter discussion title...']").type("Test Discussion 100");
+    cy.contains("button", "Create").click();
+    cy.get("form").within(() => {
+      cy.get("textarea").type("This is a test message 100!");
+      cy.get('button[type="submit"]').click();
+    });
+
+      cy.intercept("POST", "https://careapi.ohc.network/api/v1/auth/logout/").as("logoutRequest");
```

ESLint

[error] Replace `··cy.intercept("POST",·"https://careapi.ohc.network/api/v1/auth/logout/").as("logoutRequest"` with `cy.intercept("POST",·"https://careapi.ohc.network/api/v1/auth/logout/").as(⏎······"logoutRequest",⏎····`

(prettier/prettier)